src/piptool.py

```diff
@@ -189,7 +189,12 @@ def main():
     for req in reqs:
         name, version, extras = as_tuple(req)
         repo_name = repository_name(args.repo_prefix, name, version, python_version)
-        whl_targets["%s" % name] = "@%s//:pkg" % repo_name
+        if req in overrides:
```

Comment:
would you please avoid using `continue`

Reply:
what would you prefer?

a) two for loops, each over a subset of reqs (each subset created via a filter call?)
b) moving the lines 198-213 in 1 indentation and making the else clause longer?

Reply:
i think b is good enough